bioptim/interfaces/acados_interface.py

```diff
@@ -166,9 +166,118 @@ def __set_costs(self, ocp):
         self.mayer_costs = SX()
         self.W = np.zeros((0, 0))
         self.W_e = np.zeros((0, 0))
+        ctrl_objs = [
+            "TRACK_TORQUE",
+            "MINIMIZE_TORQUE",
+            "MINIMIZE_MUSCLES_CONTROL",
+            "TRACK_MUSCLES_CONTROL",
+            "MINIMIZE_ALL_CONTROLS",
+            "TRACK_ALL_CONTROLS",
+        ]
```

Comment:
Instead of comparing text, I suggest to compare the types. This is slightly faster and will not be prone to error if the names change. Also, by design TRACK_TORQUE and MINIMIZE_TORQUE are the same type. This would therefore reduce the number of tests